app/src/main/res/values/themes.xml

```diff
@@ -52,6 +52,12 @@
     <item name="android:textColorPrimary">@color/component_color_shared_primary_dark_text_color</item>
   </style>
 
+  <style name="DeprecationAlertDialogTheme" parent="OppiaAlertDialogTheme">
+    <item name="android:paddingBottom">3dp</item>
+    <item name="android:color">@color/component_color_shared_dialogs_radio_button_unchecked_color</item>
```

Comment:
This color does not appear to be named correctly yet. You can define a new component_color for it if none exists, per the darkmode guidelines.